.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 22.1.0 # must match test-requirements.txt
+    hooks:
+      - id: black
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1  # must match test-requirements.txt
```

Comment:
Should we have a small test that reads both both files and tests for consistency? I could see it being easy to miss this, especially because the comments aren't duplicated in `test-requirements.txt` which is the first place someone will look.

Reply:
I might need a hand with that...

Reply:
The comment is in both now.